nexus_constructor/stream_fields_widget.py

```diff
@@ -337,24 +339,26 @@ def _record_advanced_f142_values(self, stream: F142Stream):
         Save the advanced f142 properties to the stream data object.
         :param stream: The stream data object to be modified.
         """
-        raise NotImplementedError
+        stream.adc_pulse_debug = self.f142_adc_pulse_debug_checkbox.isChecked()
```

Comment:
`adc_pulse_debug` is not in F142 schema. It is only for ev42.

https://github.com/ess-dmsc/kafka-to-nexus/blob/main/documentation/writer_module_f142_log_data.md